### ISSUE TITLE
Doesnt work

### ISSUE THREAD

**Issue by KenWa51** [theusaf/kahoot.js-updated#2]
Bot doesnt join the kahoot?
```//Int

var Kahoot = require("kahoot.js-updated");

console.log("[CA] Kahoot Killer Started...");

//Settings

const BotAmount = 10;
const GamePin = 411230;
const NameList = [""]

///Setting up Variables

var clients = [];

//Making Clients

for(i=0;i<BotAmount;i++){
    
    clients.push(new Kahoot);
    
};

console.log(clients[1])

//Making Clients Join

for (i = 0; i < clients.length; i++) { 
    
    const client = clients[i]
    
    setTimeout(function(){ //One second between joins
        
        client.join(GamePin, NameList[i]);
        console.log("guy joined")
        
    }, 1000*i);
    
}`

**Comment by KenWa51** [comment]
Example doesnt work either exits with the following: `                var l = ((new Date).getTime() - packet.ext.timesync.tc - packet.ext.timesync.p) / 2;
                                                                    ^

TypeError: Cannot read property 'tc' of undefined
    at WSHandler.getPacket (C:\Users\kenny\node_modules\kahoot.js-updated\src\WSHandler.js:135:55)
    at WSHandler.message (C:\Users\kenny\node_modules\kahoot.js-updated\src\WSHandler.js:237:30)
    at WebSocket.WSHandler.ws.on.msg (C:\Users\kenny\node_modules\kahoot.js-updated\src\WSHandler.js:28:7)
    at WebSocket.emit (events.js:189:13)
    at Receiver.receiverOnMessage (C:\Users\kenny\node_modules\kahoot.js-updated\node_modules\ws\lib\websocket.js:789:20)
    at Receiver.emit (events.js:189:13)
    at Receiver.dataMessage (C:\Users\kenny\node_modules\kahoot.js-updated\node_modules\ws\lib\receiver.js:422:14)
    at perMessageDeflate.decompress (C:\Users\kenny\node_modules\kahoot.js-updated\node_modules\ws\lib\receiver.js:379:23)
    at _decompress (C:\Users\kenny\node_modules\kahoot.js-updated\node_modules\ws\lib\permessage-deflate.js:298:9)
    at _inflate.flush (C:\Users\kenny\node_modules\kahoot.js-updated\node_modules\ws\lib\permessage-deflate.js:376:7)`

**Comment by theusaf** [comment]
Confirmed. Kahoot recently changed its API. A fix will be coming soon.

**Comment by theusaf** [comment]
Fixed in [d52253c](https://github.com/theusaf/kahoot.js-updated/commit/d52253cd8c997e253221d2c0f98119f51d449ff2)

However, it wasn't intensely tested and there may be bugs. Feel free to comment if you find more issues.
